app/src/main/java/org/p2p/wallet/svl/model/SendLinkGenerator.kt

```diff
@@ -37,8 +43,13 @@ object SendLinkGenerator {
     private fun generateSymbols(): List<String> {
         val regex = Regex(REGEX_LINK_ALLOWED_SYMBOLS)
         val random = Random.Default
-        return (1..SYMBOLS_COUNT)
-            .map { random.nextInt(ASCII_CHARACTERS_COUNT).toString() }
```

Comment:
random.nextInt(ASCII_CHARACTERS_COUNT).toString() returned Int as a String. To convert ASCII Int to Char we user Char